lib/winrm-elevated/runner.rb

```diff
@@ -26,6 +26,7 @@ class Runner
       # @param [WinRMWebService] WinRM web service client
       def initialize(winrm_service)
```

Comment:
Should we just change this to take an executor or somehow otherwise change this API? The reason being I believe we scope the executor and do cleanup when done, but here we're creating it and never destroying it.